lib/philomena/duplicate_reports/search_query.ex

```diff
@@ -21,7 +21,7 @@ defmodule Philomena.DuplicateReports.SearchQuery do
     search_query
     |> cast(attrs, [:distance, :limit])
     |> validate_number(:distance, greater_than_or_equal_to: 0, less_than_or_equal_to: 1)
-    |> validate_number(:limit, greater_than_or_equal_to: 1, less_than_or_equal_to: 50)
+    |> validate_number(:limit, greater_than_or_equal_to: 1, less_than_or_equal_to: 100)
```

Comment:
100 isn't allowed as a limit anywhere else, is it?

Reply:
the main issue is that it doesn't sort the results so 50 may not be enough especially with a distance of 1